var/spack/repos/builtin/packages/linaro-forge/package.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import platform
+import subprocess
+
+from spack.package import *
+
+
+class LinaroForge(Package):
+    """Build reliable and optimized code for the right results on multiple
+    Server and HPC architectures, from the latest compilers and C++ standards
+    to Intel, 64-bit Arm, AMD, OpenPOWER and Nvidia GPU hardware. Linaro Forge
+    combines Linaro DDT, the leading debugger for time-saving high performance
+    application debugging, Linaro MAP, the trusted performance profiler for
+    invaluable optimization advice across native and Python HPC codes, and
+    Linaro Performance Reports for advanced reporting capabilities."""
+
+    homepage = "https://www.linaroforge.com"
+    maintainers("kenche-linaro")
+
+    if platform.machine() in ["aarch64", "arm64"]:
+        version(
+            "23.0.3", sha256="a7e23ef2a187f8e2d6a6692cafb931c9bb614abf58e45ea9c2287191c4c44f02"
+        )
+        version(
+            "23.0.2", sha256="698fda8f7cc05a06909e5dcc50b9956f94135d7b12e84ffb21999a5b45c70c74"
+        )
+        version(
+            "23.0.1", sha256="552e4a3f408ed4eb5f1bfbb83c94530ee8733579c56c3e98050c0ad2d43eb433"
+        )
+        version("23.0", sha256="7ae20bb27d539751d1776d1e09a65dcce821fc6a75f924675439f791261783fb")
+        version(
+            "22.1.4", sha256="4e2af481a37b4c99dba0de6fac75ac945316955fc4170d06e321530adea7ac9f"
+        )
+        version(
+            "21.1.3", sha256="4a4ff7372aad5a31fc9e18b7b6c493691ab37d8d44a3158584e62d1ab82b0eeb"
+        )
+    elif platform.machine() == "ppc64le":
+        version(
+            "23.0.3", sha256="5ff9770f4bc4a2df4bac8a2544a9d6bad9fba2556420fa2e659e5c21e741caf7"
+        )
+        version(
+            "23.0.2", sha256="181b157bdfc8609b49addf63023f920ebb609dbc9a126e9dc26605188b756ff0"
+        )
+        version(
+            "23.0.1", sha256="08cffef2195ea96872d56e827f320eed40aaa82fd3b62d4c661a598fb2fb3a47"
+        )
+        version("23.0", sha256="0962c7e0da0f450cf6daffe1156e1f59e02c9f643df458ec8458527afcde5b4d")
+        version(
+            "22.1.3", sha256="6479c3a4ae6ce6648c37594eb0266161f06d9f89010fca9299855848661dda49"
+        )
+        version(
+            "22.0.4", sha256="f4cb5bcbaa67f9209299fe4653186a2829760b8b16a2883913aa43766375b04c"
```

Comment:
If you're supporting `22.1.4` for `aarch64` and `arm64`, do you intend to support `22.1.4` (or only `22.0.4`) for `ppc64le` and `x86_64`?

Bottom line: Are you deliberately supporting `22.1.4` for those machines and `22.0.4` for the other or should all machines support the same version(s)?

Reply:
We deliberately released a minor update (`22.1.4`) for the `aarch64/arm64` version. The latest `22.1.x` version for `x86_64` and `ppc64le` is `22.1.3`.

Reply:
Great!  Thanks for the clarification.